src/components/Button/Button.js

```diff
@@ -1,11 +1,29 @@
 import React from 'react'
+import cx from 'classnames'
 import './Button.module.css'
 
-const BUTTON_LABEL = 'some domain specific button'
+function Button ({
+  disabled = false,
+  wide = false,
+  variant = 'secondary',
+  className,
+  onClick = () => {},
+  children
+}) {
+  const classes = cx('button', {
+    wide,
+    primary: (variant === 'primary'),
```

Comment:
at first I thought this was some fancy destructuring syntax and was like "woah".

I think I like this style

`<Button mini>blah</Button>`

more than

`<Button variant='mini'>blah</Button>`

Can see arguments for both.

Reply:
Your suggestion looks nicer, but doesn't prevent us from setting both `primary` and `secondary` options, which would break the design.
If there aren't any strong opinions, I'd stick to single prop (wide is a switch and not a variant).

Reply:
This makes sense.

hmm. so 'wide' is a switch, but 'mini' is a variant? This seems like a gotcha.

Reply:
Yeah. I mean mini might be a wrong name for a variant here, but I know the `wide` modifier is quite common with buttons, and making separate variants for it would be cumbersome.